package-overrides/npm/functify@0.4.0.json

```diff
@@ -0,0 +1,3 @@
+{
+  "main": "src/functify"
```

Comment:
I think for this ES main to work in jspm 0.16 might require `jspmNodeConversion: false` as well here? Let me know.